internal/cluster/transition.go

```diff
@@ -542,7 +542,12 @@ func (th *transitionHandler) PostPreparingTimedOut(sw stateswitch.StateSwitch, a
 		return err
 	}
 
-	eventgen.SendInstallationPreparingTimedOutEvent(params.ctx, params.eventHandler, *cluster.ID)
+	causedBy := "Preparation of configuration for installation of the cluster was started but did not complete. In the context of a timeout this is most likely to indicate that there was a problem with an assisted-service pod, please retry the installation."
```

Comment:
suggesting to consult with UX to make the message as clear as possible @nirfarkas

Reply:
The message should clearly state:
What just went wrong?
Why did this happen?
How to solve it?

Reply:
In terms of why, we can't say for certain that the assisted pod crashed, only that for whatever reason the async goroutine to perform preparation did not complete.

I think the phrases we have suggested encapsulate this, or would you suggest alternative wording?

Reply:
I'm going with 

"An unexpected error occurred in the assisted installer, please retry installation and contact support if the issue persists"

and 

"One or more cluster hosts were not prepared successfully, please check host events for more information"